Source/GUI/MemWatcher/MemWatchModel.cpp

```diff
@@ -624,14 +622,17 @@ void MemWatchModel::sortRecursive(int column, Qt::SortOrder order, MemWatchTreeN
   }
 }
 
-QString MemWatchModel::getAddressString(u32 address, bool isPointer) const
+QString MemWatchModel::getAddressString(const MemWatchEntry* entry) const
 {
   std::stringstream ss;
-  if (isPointer)
+  if (entry->isBoundToPointer())
   {
-    ss << "P->" << std::hex << std::uppercase << address;
+    int level = static_cast<int>(entry->getPointerLevel());
```

Comment:
What was the resolution on the discussion on static_cast here?

Reply:
This is separate to the discussion on Discord, where @cristian64 pointed out int-to-float conversions cannot use `static_cast` due to aliasing. However, in this scenario we are casting from `size_t` to `int` which is safe and is used various other times in `DlgAddWatchEntry.cpp` when retrieving the pointer level.